### DISCUSSION TITLE
No Need of two Login

### DISCUSSION THREAD

**Discussion by fabo123456** [lockerpm/.github#10] [Ideas]
Hello everyone

I really like the password manager. The only thing that bothers me is the requirement that you have to log in twice. So in addition to the master password, I have a second one that I have to remember :(
Isn't it possible to remove this? The second logins can remain in the case for any changes to personal data. But it should be separated to the password-manager.

**Comment by Joey344532** [comment]
Hi @fabo123456, thanks for reaching out. In fact, both passwords are needed, as the first one is used to authenticate yourself in our system (this account can be used not only for Locker, but also the other products in our ecosystem, which you can find [here](https://cystack.net/products)). And the second but most important password is the Master Password used for encrypting and decrypting data. 

We understand that the two passwords might be complex for users. That is why we have made a huge improvement in our login flow: now you can use a passkey or email verification to authenticate yourself, instead of using the first password. We expect this improvement to lead to a more seamless, less-thing-to-remember flow. Please try and give us your insightful feedback :)
